exonum-java-binding/package_app.sh

```diff
@@ -6,6 +6,21 @@
 # Fail immediately in case of errors and/or unset variables
 set -eu -o pipefail
 
+function build-ejb-app-macos() {
+    # Set RUSTFLAGS to adjust RUNPATH of the binary.
+    export RUSTFLAGS="-C link-arg=-Wl,-rpath,@executable_path/lib/native"
+    export RUSTFLAGS="${RUSTFLAGS} -C link-arg=-Wl,-rpath,@executable_path/lib/native"
```

Comment:
I am not sure I see the difference between the two lines above :confused:

Reply:
Ouch, artifact from previous tries